packages/synapse-react-client/src/style/components/_query-wrapper-text-input.scss

```diff
@@ -1,7 +1,6 @@
 @use '../abstracts/variables' as SRC;
 
-.QueryWrapperSearchInput,
-.QueryWrapperSqlEditorInput {
+.QueryWrapperSearchInput {
```

Comment:
These styles now only apply to the regular (per-column) search component, which is a little more complicated so I didn't touch it for now